showcase.json

```diff
@@ -2853,5 +2853,25 @@
   "twitter": "https://twitter.com/221BravoApp",
   "youtube": "",
   "submittableId": "36556639"
+},
+  {
+  "name": "Tali AI ",
```

Comment:
Can you please remove the extra space after AI?

Reply:
This is done.